recipes/rvlib/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "rvlib" %}
+{% set version = "0.0.1" %}
+{% set sha1 = "d74f5c10e956c09349a22a1a80e7e225978de253" %} 
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantEcon/rvlib/archive/{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+
+build:
+  skip: True  # [win or not py35]
```

Comment:
in [this](https://github.com/conda-forge/staged-recipes/pull/1163#issuecomment-237248912) comment @patricksnape  suggested restricting ourselves to Python 3.5 due to the use of a specific tuple unpacking.